app/models/ticket_purchase.rb

```diff
@@ -53,4 +53,10 @@ def self.update_quantity(conference, quantity, ticket, user)
     purchase.quantity = quantity if quantity > 0
     purchase
   end
+
+  def self.tickets_sold(ticket_id, conference_id)
```

Comment:
This method also belongs to ticket and probably you can do the inject in a database query for better performance ...